core/test/martian/openapi_test.cljc

```diff
@@ -102,8 +102,8 @@
 (deftest jira-openapi-v3-test
   (is (= 410
          (-> jira-openapi-v3-json
-             (openapi->handlers {:encodes ["json"]
-                                 :decodes ["json"]})
+             (openapi->handlers {:encodes ["application/json"]
+                                 :decodes ["application/json"]})
```

Comment:
This doesn't affect the thing under test, but gets rid of some false-negative warnings in the test logs.